ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -117,8 +119,9 @@ data Cmd
     | Close
     | ReOpen
     | GetBlock BlockId
+    | GetHeader BlockId
     | GetBlockIds
-    | PutBlock BlockId Predecessor
+    | PutBlock BlockId Predecessor ByteString
```

Comment:
I just realised that this `ByteString` is in fact the header. Can you document that?